buster/chatbot.py

```diff
@@ -247,6 +247,9 @@ def process_input(self, question: str) -> str:
         Main function to process the input question and generate a formatted output.
         """
 
+        # We make sure there is always a newline at the end of the question to avoid completing the question.
+        if not question.endswith("\n"):
+            question += "\n"
         logger.info(f"User Question:\n{question}")
```

Comment:
I think it's better to log whatever the user actually entered:
```suggestion
        logger.info(f"User Question:\n{question}")
        
        # We make sure there is always a newline at the end of the question to avoid completing the question.
        if not question.endswith("\n"):
            question += "\n"
```